pbs_light.go

```diff
@@ -142,6 +143,64 @@ func getAccountMetrics(id string) *AccountMetrics {
 	return am
 }
 
+type cookieSyncRequest struct {
+	UUID    string   `json:"uuid"`
+	Bidders []string `json"bidders"`
+}
+
+type cookieSyncResponse struct {
+	UUID         string           `json:"uuid"`
+	Status       string           `json:"status"`
+	BidderStatus []*pbs.PBSBidder `json:"bidder_status"`
+}
+
+func cookieSync(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
```

Comment:
w and r seem pretty typical in all the example code I've seen, but open to other names. Open to something else if somebody who's more go-proficient has a better suggestion.

In regard to splitting into separate files... again, not sure whether there's a best practice here, but I'm reluctant to create even more subpackages (like a "handlers") dir. Right now there are handlers in the pbs/usersync.go code, and that was complicated because there are cross-dependencies between the pbs package and others. It's easy enough to flip these out later, but open to suggestions.